kolibri/plugins/learn/assets/src/views/ExamPage/index.vue

```diff
@@ -5,168 +5,251 @@
-              <KButton
-                :disabled="questionNumber === exam.question_count - 1"
-                :primary="true"
-                :dir="layoutDirReset"
-                :aria-label="$tr('nextQuestion')"
-                :appearanceOverrides="navigationButtonStyle"
-                @click="goToQuestion(questionNumber + 1)"
-              >
-                <span v-if="displayNavigationButtonLabel">{{ $tr('nextQuestion') }}</span>
-                <template #iconAfter>
-                  <KIcon
-                    icon="forward"
-                    :color="$themeTokens.textInverted"
-                    :style="navigationIconStyleNext"
-                  />
-                </template>
-              </KButton>
-              <KButton
-                :disabled="questionNumber === 0"
-                :primary="true"
-                :dir="layoutDirReset"
-                :appearanceOverrides="navigationButtonStyle"
-                :class="{ 'left-align': windowIsSmall }"
-                :aria-label="$tr('previousQuestion')"
-                @click="goToQuestion(questionNumber - 1)"
+    <div v-else>
+      <KGrid :gridStyle="gridStyle">
+        <!-- this.$refs.questionListWrapper is referenced inside AnswerHistory for scrolling -->
+        <KGridItem
+          v-if="windowIsLarge"
+          ref="questionListWrapper"
+          :layout12="{ span: 4 }"
+          class="column-pane"
+        >
+          <div class="column-contents-wrapper">
+
+            <KPageContainer>
+              <div
+                v-for="(section,index) in exam.question_sources"
+                :key="index"
               >
-                <template #icon>
-                  <KIcon
-                    icon="back"
-                    :color="$themeTokens.textInverted"
-                    :style="navigationIconStylePrevious"
-                  />
-                </template>
-                <span v-if="displayNavigationButtonLabel">{{ $tr('previousQuestion') }}</span>
-              </KButton>
-            </component>
+                <div
+                  class="spacing-items"
+                  :style="{
+                    backgroundColor: $themePalette.grey.v_100,
+                  }"
+                >
+                  <KGrid>
+                    <KGridItem
+                      :layout12="{ span: 8 }"
+                      :layout8="{ span: 6 }"
+                      :layout4="{ span: 3 }"
+                    >
+                      <span
+                        class="quiz-title"
+                      >
+                        {{ section.section_title }}
+                      </span>
+                    </KGridItem>
+
+                    <KGridItem
+                      :layout12="{ span: 4 }"
+                      :layout8="{ span: 2 }"
+                      :layout4="{ span: 1 }"
+                    >
+                      <div style="float:right">
```

Comment:
For this chevron you should be able to use `text-align: right` here rather than the `float` property. We typically avoid `float` because it is not intended for use as a general layout property but more to position things within areas of text.